frontend/src/app/components/add-post-modal/add-post.component.ts

```diff
@@ -61,6 +70,10 @@ export class AddPostComponent {
     );
   }
 
+  // ngAfterViewInit(): void {
```

Comment:
Should we remove this commented out section?

Reply:
Ye, thats my bad i was trying some different ways and forgot to remove this.